gdnative-core/src/macros.rs

```diff
@@ -88,7 +85,8 @@ macro_rules! godot_gdnative_terminate {
                 return;
             }
 
-            let __result = ::std::panic::catch_unwind(|| $callback(options));
+            let in_editor = (*options).in_editor;
+            let __result = ::std::panic::catch_unwind(|| $callback(in_editor));
```

Comment:
Didn't wrap the `in_editor: bool` in a new type. Would we want a new type to keep future code from breaking here? I don't have a sense if `sys::godot_gdnative_terminate_options` would ever change.

Reply:
I think it makes sense to create a struct here even if it's just one trait. Note that the `_in_editor` argument name won't be shown to users. Instead they will only see a `bool` and have no idea what it might mean.

Reply:
Updated. Renamed `InitInfo` to `InitializeInfo` so that it matches `TerminateInfo`. `TermInfo` felt like it conveyed the wrong idea, so it was avoided.